src/main/java/bibi/Ui.java

```diff
@@ -0,0 +1,295 @@
+package bibi;
+
+import java.io.IOException;
```

Comment:
Try to separate your import statements into packages that are related, and try to order them alphabetically.